recipe/meta.yaml

```diff
@@ -289,6 +283,7 @@ outputs:
 
   - name: ray-tune
     build:
+      skip: true
```

Comment:
about section:
home url should be: https://www.ray.io/
doc url should be: https://docs.ray.io/en/latest/